sdk/include/opentelemetry/sdk/trace/span_data.h

```diff
@@ -186,9 +190,12 @@ class SpanData final : public Recordable
     attribute_map_.SetAttribute(key, value);
   }
 
-  void AddEvent(nostd::string_view name,
-                core::SystemTimestamp timestamp,
-                const opentelemetry::common::KeyValueIterable &attributes) noexcept override
+  void AddEvent(
```

Comment:
to make consistent with ThreadsafeSpanData and avoid having to explicitly have to reference `.opentelemetry::sdk::trace::Recordable::AddEvent`